vkcs/blockstorage/resource_volume.go

```diff
@@ -99,8 +101,7 @@ func ResourceBlockStorageVolume() *schema.Resource {
 				},
 				Optional:    true,
 				ForceNew:    false,
-				Computed:    true,
-				Description: "Map of key-value metadata of the volume.",
+				Description: "Map of key-value metadata of the volume, which is set in terraform config. Terraform will show the difference only in the metadata that is specified in the terraform config. If some metadata is added from the outside by the user, it will be deleted on the next apply.",
```

Comment:
Key-value map to configure metadata of the volume. _note_ Changes to keys that are not in scope, i.e. not configured here, will not be reflected in planned changes, if any, so those keys can be "silently" removed during an update.